src/ar_http_iface_cowboy_handler.erl

```diff
@@ -691,7 +815,7 @@ arweave_peer(Req) ->
 split_path(Path) ->
 	lists:filter(
 		fun(C) -> C /= <<>> end,
-		string:split(Path, <<"/">>, all)
+		binary:split(Path, <<"/">>, [global])
```

Comment:
Did you sneak in a bugfix here? :) Do you know what endpoints are affected of this?

Reply:
Nope 😅 I think I used another split somewhere else while developing this feature and when I wrote the new split, I realised that both the `string` module and the `binary` module have their own split functions. Looking into the difference, I saw that `string:split` operates on `unicode:chardata()`, whereas `binary:split` is a BIF that only operates on binaries. Therefore I went for the function in `binary` and updated the code in this line to be consistent. I think I later got rid of the new `split` call.